chainer/function_hooks/timer.py

```diff
@@ -132,11 +133,38 @@ def _humanized_time(self, second):
             second *= 1000.0
         return '%.2f%s' % (second, 'ns')
 
-    def print_report(self, file=sys.stdout):
-        """Prints a summary report of time profiling in functions."""
+    def _align(self, second):
```

Comment:
Now I feel this method name is misleading. How about, e.g., `_choose_unit`?

Reply:
That's good. I did it.